test/Microsoft.Framework.Logging.Test/LoggerFactoryExtensionsTest.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Framework.Logging.Test
     public class LoggerFactoryExtensionsTest
     {
 #if ASPNET50
```

Comment:
The project.json does not have an entry for `aspnetcore50`. Our other repos that use Moq don't have them either. Should I remove the `ifdef` altogether since it won't be compiled for `aspnetcore50` anyway ?